pkg/doc/jsonld/embed_contexts/did_v0.11.jsonld

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
let's make it clear that these are third_party files by having third_party in the path.

e.g., pkg/doc/jsonld/embed_contexts/third_party/...

Reply:
@troyronda plz check if you are ok with the following structure:

pkg/doc/jsonld/third_party/w3.org/
* https://www.w3.org/2018/credentials/v1
* https://www.w3.org/2018/credentials/examples/v1
* https://www.w3.org/ns/did/v1
* https://www.w3.org/ns/odrl.jsonld

pkg/doc/jsonld/third_party/w3id.org/
* https://w3id.org/security/v1
* https://w3id.org/security/v2
* https://w3id.org/security/jws/v1
* https://w3id.org/security/bbs/v1
* https://w3id.org/did/v0.11
* https://w3id.org/did/v1
* https://w3id.org/citizenship/v1

pkg/doc/jsonld/third_party/dif/
* https://identity.foundation/presentation-exchange/submission/v1

pkg/doc/jsonld/third_party/trustbloc/
* https://trustbloc.github.io/context/vc/credentials-v1.jsonld
* https://trustbloc.github.io/context/vc/examples-v1.jsonld

Should `trustbloc` stay here as well?

Reply:
use the underlying hosting domain for the directory. Note w3id.org is not the hosting domain.

Example: 
w3c-ccg.github.io

Reply:
https://trustbloc.github.io/context/vc/credentials-v1.jsonld looks like https://w3id.org/security/jws/v1. Should update the examples to not even use https://trustbloc.github.io/context/vc/credentials-v1.jsonld

Reply:
It would also be preferable to see what test is using https://trustbloc.github.io/context/vc/examples-v1.jsonld and see if it can be updated to not use it.

Reply:
👍